de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc

```diff
@@ -1232,6 +1232,112 @@ image:chapterModeling/MaturityOverview.png[Maturity overview]
+
+Now you can add a state machine to the component `CAM` in the _Navigator_ with `Context` › `State Machines` › `Add StateMachine`.
+
+|image:chapterStateMachines/4.png[OpenDiagramEditor]
+
+|{counter:step1}.
+|Having created a _State Machine_ for the camera, now open the _State Machine Diagram Editor_ in the _Navigator_ with `Context` › `Open Diagram Editor`.
+|image:chapterStateMachines/5.png[AddStateMachine]
+
+|{counter:step1}.
+|The _Diagram Editor_ shall look as shown in the image. 
+|image:chapterStateMachines/DiagramEditor.png[DiagramEditor]
+
+|{counter:step1}.
+|Now to add a _State_ (same for other objects in the _Diagram Palette_, i.e. `State`, `Transition`, `ForbidsConstraint`, ...), left click on the object in the Palette once and then left click again on the position in the diagram where you want to add it.
+
+The state machine diagram for the `CAM` object shall look as shown in the image.
+|image:chapterStateMachines/AddState.png[AddState] 
+
+|{counter:step1}.
+|After creating a transition connection in the Diagram Editor, you want to create a _TransitionTriggerEvent_ and assign it to the previously created transition. 
+
+Go in the _Editor_ using _SM_ instance in the _Navigator_: `Context` › `Open Editor` or simply by double clicking on the _SM_ instance in the _Navigator_.
+
+In the _Editor_ if you press [Add TransitionTriggerEvent] button, a new trigger event is created. 
+
+Name it under the column `Name`, and give it a description under the `detail`.
+
+|image:chapterStateMachines/Open Editor.png[Open Editor] image:chapterStateMachines/AddTransitionTriggerEvent.png[AddTransitionTriggerEvent]
+
+|{counter:step1}.
+|Now assign the created trigger event to the `Transition` you created in the diagram. Under `trigger` column in the `transitions` section, you can choose a trigger action from the list as shown in the image.
+
+NOTE: Transitions can be created or removed in the _Diagram Editor_, but also manually in the normal _Editor_.
+
+|image:chapterStateMachines/AssigningTriggers.png[AssigningTriggers] image:chapterStateMachines/OpenedWizardForTriggerActions.png[OpenedWizardForTriggerActions]
+
+|{counter:step1}.
+|After assigning, go back to the _Diagram Editor_ and update the state machine diagram using the yellow button up right the _State Machine_ block, then you will be able to see the new transition.
+|image:chapterStateMachines/UpdateStateMachine.png[UpdateStateMachine]
+
+|{counter:step1}.
+|Now having created the various components of the system and described their behavior using state machines, the _State Machine_ for the `AOCS` system can be created in a newly created _ElementConfiguration_ namely `CONOPS` (standing for concept operations). This state machine shall describe the behavior of the whole system.

```

Comment:
We shouldnt use "CONOPS" in the example. Try to find a different name here. The reason is that CONOPS is a critical term for export control and might cause issues.

Reply:
changed